packages/visualizations/src/components/Chart/Chart.svelte

```diff
@@ -185,29 +198,47 @@
             {#if options.description}
                 <p id={chartId} class="a11y-invisible-description">{options.description}</p>
             {/if}
-        </div>
+            {#if options?.legend?.custom}
+                <figcaption>
+                    <CategoryLegend {legendOptions} />
+                </figcaption>
+            {/if}
+        </figure>
         {#if options.source}
             <SourceLink source={options.source} />
         {/if}
```

Comment:
The legend and view source looks a bit too close IMO, what do you think ? 
![Capture d’écran 2023-05-22 à 17 36 08](https://github.com/opendatasoft/ods-dataviz-sdk/assets/84501475/24296d4e-cd1f-4217-83ea-b1678a30a75b)